webrtc/videoroom/lib/videoroom_web/room_channel.ex

```diff
@@ -200,4 +180,28 @@ defmodule VideoRoomWeb.RoomChannel do
   defp send_to_pipeline(socket, message) do
     socket.assigns.pipeline |> send(message)
   end
+
+  defp camelizeKeys(%{} = map) do
+    map
+    |> Map.new(fn {key, value} ->
+      camelized_key =
+        key
+        |> Atom.to_string()
+        |> Macro.camelize()
+
+      key_prefix =
+        camelized_key
+        |> String.at(0)
+        |> String.downcase()
+
+      key_sufix =
+        camelized_key
+        |> String.slice(1..-1)
+
+      {
+        key_prefix <> key_sufix,
+        value
+      }
+    end)
+  end
 end
```

Comment:
Wouldn't be it better to include in this function this piece of code
https://github.com/membraneframework/membrane_demo/blob/ef84c736c0d56120763f068bfa27625c3cf3566b/webrtc/videoroom/lib/videoroom_web/room_channel.ex#L107-L116

and rename it to e.g. prepareJson or serialize etc.